### ISSUE TITLE
Few questions

### ISSUE THREAD

**Issue by sytelus** [ddlee96/awesome_cnn#2]
Hi, thank you for the awesome work! I'm planning to add few more features and hope to contribute back. Before I start work, I had few questions:

1. Is this code upgraded to Pytorch 0.4 and tested with it? 
2. Do you have plans to support Tensorboard?
3. Were you able to reproduce results for validation accuracy in papers for ImageNet?

Thanks!

**Comment by ddlee-cn** [comment]
@sytelus Thanks for your feedback.
Actualy, this repo is a collection of code examples. It provides readable codes(in PyTorch) and visualization of network structure(with NetScope). So I'm not gooing to provide reproduced results either futher support for updated frameworks. For that, you may refer to https://github.com/kuangliu/pytorch-cifar
